browser/resources/settings/ad_block_only_mode_page/ad_block_only_mode_page.html

```diff
@@ -0,0 +1,25 @@
+<style include="settings-shared md-select">
+  .description {
+    padding: 1em 0;
```

Comment:
Hey sorry, is this intentional? Normally we avoid `em` units in Brave in favor of pixel units. Possibly `var(--leo-spacing-m)` but I'm not sure.